examples/bpk-component-content-cards/stories.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { storiesOf } from '@storybook/react';
+
+import {
+  ExampleWithOneCard,
+  ExampleWithTwoCards,
+  ExampleWithThreeCards,
+} from './examples';
+
+storiesOf('bpk-component-content-cards', module)
+  .add('Default One Card', ExampleWithOneCard)
+  .add('Default Two Cards', ExampleWithTwoCards)
+  .add('Default Three Cards', ExampleWithThreeCards);
```

Comment:
Can you add an additional option here for visual tests for our snapshot testing? (Currently disabled but for when we reenable it)

An example can be found here: https://github.com/Skyscanner/backpack/blob/main/examples/bpk-component-button/stories.js#L68

Here its the naming of the story needs to be as per the above link so its captured correctly

Reply:
Ok, like this? :) 
<img width="387" alt="Screenshot 2022-08-19 at 15 54 53" src="https://user-images.githubusercontent.com/58072716/185634374-b249d146-fffb-4e3c-82c1-b0c5e8e667a6.png">

Reply:
You can just create the one visual test and just use the `ExampleWithThreeCards` for this!

Reply:
Ah so just like this? :)
<img width="391" alt="Screenshot 2022-08-23 at 10 34 02" src="https://user-images.githubusercontent.com/58072716/186112059-4ff9dc63-8b06-4b5f-9011-72286138cea8.png">

Reply:
Yep that's the one :)